data/mods/Xedra_Evolved/effects/effects.json

```diff
@@ -1033,15 +1033,13 @@
     "rating": "mixed"
   },
   {
-    "type": "mutation",
-    "id": "karma_arms",
-    "name": { "str": "Karma" },
-    "points": 1,
-    "visibility": 10,
-    "description": "Ethereal hands appear behind you back, making you looks like an ancient god.",
-    "valid": false,
-    "starting_trait": false,
-    "purifiable": false,
+    "type": "effect_type",
+    "id": "spell_karma_arms_eff",
+    "name": [ "Karma" ],
+    "desc": [ "Ethereal hands appear behind you back, making you looks like an ancient god" ],
```

Comment:
you -> your .  End with a period.

Reply:
to make a suggestion, you can press this small button, and edit the text inside
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/67688115/1755c72a-105e-4524-bd4f-fbfe6cae71a2)

```suggestion
    "desc": [ "Ethereal hands appear behind your back, making you looks like an ancient god." ],
```
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/67688115/954f5c64-8400-4e51-89e7-43b9e82a7970)